pkg/api/openshiftcluster.go

```diff
@@ -27,6 +27,11 @@ type OpenShiftCluster struct {
 	Lock sync.Mutex `json:"-"`
 }
 
+// IsWimi checks whether a cluster is a Workload Identity cluster or Service Principal cluster
+func (oc *OpenShiftCluster) IsWorkloadIdentity() bool {
```

Comment:
nit: This is probably a matter of preference, but I like `IsWimi` better than `IsWorkloadIdentity` because "the cluster is wimi" sounds more natural than "the cluster is workload identity".
If it's only me, please ignore this comment.

Reply:
sorry I missed the previous discussion.

Reply:
`UsesWorkloadIdentity` might be better if we're worried about reading it like that - my comment was moreso that asking everyone to go learn more acronyms is not ideal!

Reply:
Agree. UsesWorkloadIdentity is probably the best option. 🙂

Reply:
UsesWorkloadIdentity sounds good to me! Thank you for the suggestion.